### ISSUE TITLE
Why is water diggable?

### ISSUE THREAD

**Issue by BF5258** [PrismarineJS/minecraft-data#926]
In versions 1.19 and 1.20 water and lava are marked as diggable. This cause mineflayer-collect block to try to mine them.
![image](https://github.com/user-attachments/assets/1f3ae083-244a-4d49-9ef9-08cf1570f784)
My understanding is that "diggable" means able to be broken in survival by mining. If this is not the case, please explain.

**Comment by rom1504** [comment]
yes we need to agree on the definition and fix it, can you look into it?

**Comment by BF5258** [comment]
minecraft-data-generator considers any non-air block with finite hardness to be diggable. ([BlocksDataGenerator.java](https://github.com/PrismarineJS/minecraft-data-generator/blob/73cbac110a44b7056464ca440bb073fb46e8a1af/mc/1.21/src/main/java/dev/u9g/minecraftdatagenerator/generators/BlocksDataGenerator.java#L137))
![Image](https://github.com/user-attachments/assets/0c7da08c-2c97-494a-a4e1-a726ba2ab80f)
Water and lava both have a finite hardness but cannot be targeted. ([minecraft.wiki](https://minecraft.wiki/w/Breaking#Blocks_by_hardness))
![Image](https://github.com/user-attachments/assets/728941ba-e139-4b4a-b696-f11affabfdea)
Checking for water and lava as well as air should fix it, but I am unfamiliar with Java.

**Comment by BF5258** [comment]
Although they can only be mined by vanilla clients using bugs, the server seems to consider bubble columns to be mineable. They are placed back almost immediately but do cause block updates.

https://github.com/user-attachments/assets/a5ea324e-885d-4981-849e-1b2a736b28e8

Water and lava also seem to be minable but in 1.21 they are replaced by themselves, resulting in no change.
waterlogged slab -> waterlogged water + slab item
waterlogged water -> waterlogged water (no drop)
It might be possible to break liquids in versions before waterlogging was added.
